src/app/app-routing.module.ts

```diff
@@ -114,6 +115,7 @@ const routes: Routes = [
   { path: 'my-file', component: MyFileComponent, canActivate: [AuthGuard] },
   { path: 'issues', component: IssuesComponent },
   { path: 'about', component: AboutComponent },
+  { path: 'faq', component: FaqPageComponent },
```

Comment:
If you move the route into the child section of the `MainLayoutComponent`, you can omit stuff like the <app-header> in your `FaqPageComponent`